app/src/pages/common/membersPage/membersGrid/personalInfo/personalInfo.jsx

```diff
@@ -21,19 +21,19 @@ import classNames from 'classnames/bind';
 import { FormattedMessage } from 'react-intl';
 import { ADMINISTRATOR } from 'common/constants/accountRoles';
 import { userIdSelector } from 'controllers/user';
-import { projectIdSelector } from 'controllers/pages';
 import { UserAvatar } from 'pages/inside/common/userAvatar';
+import { projectKeySelector } from 'controllers/project';
```

Comment:
Should be URL alternative.

Reply:
projectKeySelector needed for backend request